### DISCUSSION TITLE
Week 3 Report

### DISCUSSION THREAD

**Discussion by derrrobin** [SE-TINF22B6/CookPit#23] [General]
Update: 
We have consulted in the team and decided to focus our website with focus on baking recipes tu. Thereby we can remove a few functions and add a few new ones.

This week we planned our workflow. We decided when we want to do the documentation, planning and programming stuff. we also created our first design with figma and thought about what our website should look like. Now we can move on to the next steps, as the rough plan for our project is in place and we can now start to build our ideas.

For the start of our next sprint, we have divided our tasks into categories in order to better assess what needs to be done now and what can be postponed to the next sprint.

The tasks for week 3 are done. We have created two design for a landing Page. Maybe you can comment which one you like the most. Your CookPit Team.

Our UML use case diagramm ✔️ 
![image](https://github.com/SE-TINF22B6/CookPit/assets/123726577/d62f115d-8b63-4b69-bd84-bd16207a1ce5)

Design 1️⃣ :
![image](https://github.com/SE-TINF22B6/CookPit/assets/123726577/8430a6ce-4657-4bb4-bc7e-085dbce2d9a2)

Design 2️⃣ : 
![image](https://github.com/SE-TINF22B6/CookPit/assets/123726577/acd5baf1-588e-4dfa-a8e7-1a2f7baa3817)

Design for our login form 3️⃣ :
![image](https://github.com/SE-TINF22B6/CookPit/assets/123726577/e5547799-162c-4266-b3fd-41f42a9af576)

Send love ❤️

**Comment by nuo-li** [comment]
As a user, may I browse receipes without an account?

**Reply by derrrobin** [reply]
The idea was, that you just need an account if you want to save a receipe or you want to upload one by your self

**Comment by Nahlam4** [comment]
I like that you have created your first use-case-diagram and your mockups. Personally, I like the first design more and I think you could change the transparency of your Login/Signup Dialog.

**Comment by LenVej** [comment]
Hi CookPit Team,
It's great to see your progress and how you've been actively planning and designing your project.
Creating two landing page designs is a fantastic start. I personally like the second design more as it looks cleaner.
Your mention of dividing tasks into categories for better assessment sounds like a smart approach.
Overall, it's evident that you're making good progress, and I’m looking forward to hearing more about your project as it develops.

Best regards,
Your CookHub Team.

**Comment by Maximon132** [comment]
As the team Time2Shine we can say that we really like your update and the progress you made. The designs look very nice, in our opinion the second design looks better, because the background doesn't take as much attention as in the first design. 
We are looking forward to see more of your project :)
